src/foobar/tests/test_views.py

```diff
@@ -67,24 +68,53 @@ def test_wallet_management(self, mock_deposit_withdrawal, mock_correction):
         self.assertEqual(response.status_code, 200)
         # Test that correction form post is correct and
         # calls function with correct params
-        response = cl.post(url,
-                           {'save_correction': ['Submit'],
-                            'balance_1': ['SEK'],
-                            'comment': ['test'],
-                            'balance_0': ['1000']})
+        cl.post(url,
+                {'save_correction': ['Submit'],
+                 'balance_1': ['SEK'],
+                 'comment': ['test'],
+                 'balance_0': ['1000']})
         mock_correction.assert_called_with(Money(1000, 'SEK'),
                                            wallet_obj.owner_id,
                                            self.user,
                                            'test')
         # Test that deposit or withdrawal form post is correct and
         # calls fucnction with correct params
-        response = cl.post(url,
-                           {'deposit_or_withdrawal_1': ['SEK'],
-                            'save_deposit': ['Submit'],
-                            'comment': ['test'],
-                            'deposit_or_withdrawal_0': ['100']})
+        cl.post(url,
+                {'deposit_or_withdrawal_1': ['SEK'],
+                 'save_deposit': ['Submit'],
+                 'comment': ['test'],
+                 'deposit_or_withdrawal_0': ['100']})
         mock_deposit_withdrawal.assert_called_with(
             Money(100, 'SEK'),
             wallet_obj.owner_id,
             self.user,
             'test')
+
+    @mock.patch('foobar.api.set_account')
+    def test_edit_profile(self, mock_set_account):
+        account_obj = factories.AccountFactory.create()
+        token = signing.dumps({'id': str(account_obj.id)})
+        url = reverse('edit_profile', kwargs={'token': token})
+        bad_token = reverse('edit_profile', kwargs={'token': 'bad'})
+        cl = self.client
```

Comment:
Any particular reason to store the test client into `cl`?

Otherwise I would argue we want to be explicit(for instance by doing `self.client.get`).